Panic while looping over the HTTP recorder headers

The following program `sample.go` triggers a panic:

```go
package main

import (
	"fmt"
	"net/http/httptest"
)

func main() {
	recorder := httptest.NewRecorder()
	recorder.Header().Add("Foo", "Bar")

	for key, value := range recorder.Header() {
		fmt.Println(key, value)
	}
}
```

Expected result:
```console
$ go run ./sample.go
Foo [Bar]
```

Got:
```console
$ yaegi ./sample.go
./simple.go:9:14: panic
reflect.Value.Call: call of nil function
goroutine 1 [running]:
runtime/debug.Stack(0x1, 0xc000148a00, 0x40)
	/Users/kevinpollet/.gvm/gos/go1.14.3/src/runtime/debug/stack.go:24 +0x9d
github.com/containous/yaegi/interp.(*Interpreter).Eval.func1(0xc000217dd8)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/interp.go:333 +0xbc
panic(0x191eae0, 0x1c35b20)
	/Users/kevinpollet/.gvm/gos/go1.14.3/src/runtime/panic.go:969 +0x166
github.com/containous/yaegi/interp.runCfg.func1(0xc0001375e0, 0xc00013b400)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/run.go:119 +0x20c
panic(0x191eae0, 0x1c35b20)
	/Users/kevinpollet/.gvm/gos/go1.14.3/src/runtime/panic.go:969 +0x166
reflect.Value.call(0xc000100900, 0xc0000107d0, 0x193, 0x1a4e27d, 0x4, 0x2262a78, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/kevinpollet/.gvm/gos/go1.14.3/src/reflect/value.go:357 +0x13d8
reflect.Value.Call(0xc000100900, 0xc0000107d0, 0x193, 0x2262a78, 0x0, 0x0, 0x1a4b840, 0x1, 0x2262a78)
	/Users/kevinpollet/.gvm/gos/go1.14.3/src/reflect/value.go:321 +0xb4
github.com/containous/yaegi/interp.callBin.func1(0xc000100900, 0xc0000107d0, 0x193, 0x2262a78, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/run.go:1077 +0x64
github.com/containous/yaegi/interp.callBin.func10(0xc0001375e0, 0xc00010b680)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/run.go:1218 +0x1c2
github.com/containous/yaegi/interp.runCfg(0xc00013b400, 0xc0001375e0)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/run.go:125 +0x6d
github.com/containous/yaegi/interp.(*Interpreter).run(0xc0001a61a0, 0xc00013ab00, 0xc000137540)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/run.go:103 +0x247
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc0001a61a0, 0xc00016e1c0, 0xda, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/kevinpollet/Documents/Dev/yaegi/interp/interp.go:416 +0x5f7
main.main()
	/Users/kevinpollet/Documents/Dev/yaegi/cmd/yaegi/yaegi.go:166 +0x6e2
```

--- a/_test/issue-775.go
+++ b/_test/issue-775.go
@@ -1,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"net/http/httptest"
+)
+
+func main() {
+	recorder := httptest.NewRecorder()
+	recorder.Header().Add("Foo", "Bar")
+
+	for key, value := range recorder.Header() {
+		fmt.Println(key, value)
+	}
+}
+
+// Output:
+// Foo [Bar]

--- a/interp/cfg.go
+++ b/interp/cfg.go
@@ -1334,7 +1334,7 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 
 		case rangeStmt:
 			if sc.rangeChanType(n) != nil {
-				n.start = n.child[1]       // Get chan
+				n.start = n.child[1].start // Get chan
 				n.child[1].tnext = n       // then go to range function
 				n.tnext = n.child[2].start // then go to range body
 				n.child[2].tnext = n       // then body go to range function (loop)
@@ -1346,7 +1346,7 @@ func (interp *Interpreter) cfg(root *node, pkgID string) ([]*node, error) {
 				} else {
 					k, o, body = n.child[0], n.child[1], n.child[2]
 				}
-				n.start = o          // Get array or map object
+				n.start = o.start    // Get array or map object
 				o.tnext = k.start    // then go to iterator init
 				k.tnext = n          // then go to range function
 				n.tnext = body.start // then go to range body

